### ISSUE TITLE
v2.0.0-beta1.1 installs CRDs in keda.k8s.io

### ISSUE THREAD

**Issue by gldraphael** [kedacore/keda#1193]
## Expected Behavior

1. `keda-operator` should start without problems.
2. CRDs should be installed in the `keda.sh` apigroup.

## Actual Behavior

1. `keda-operator` doesn't start...

	```
	$ kubectl logs pod/keda-operator-695d978ddb-jwtdf -n keda
	I0924 12:07:43.230273       1 request.go:621] Throttling request took 1.048013073s, request: GET:https://10.0.0.1:443/apis/autoscaling/v2beta2?timeout=32s
	{"level":"info","ts":1600949264.2348206,"logger":"controller-runtime.metrics","msg":"metrics server is starting to listen","addr":":8080"}
	{"level":"info","ts":1600949264.236287,"logger":"controllers.ScaledObject","msg":"Running on Kubernetes 1.18"}
	{"level":"info","ts":1600949264.2364168,"logger":"setup","msg":"Starting manager"}
	{"level":"info","ts":1600949264.2364318,"logger":"setup","msg":"KEDA Version: v2"}
	{"level":"info","ts":1600949264.2364368,"logger":"setup","msg":"KEDA Commit: 717aab81817f0d9b6d02b0e6d1499367c751faf2"}
	{"level":"info","ts":1600949264.2364426,"logger":"setup","msg":"Go Version: go1.15.1"}
	{"level":"info","ts":1600949264.236449,"logger":"setup","msg":"Go OS/Arch: linux/amd64"}
	{"level":"info","ts":1600949264.2367775,"logger":"controller-runtime.manager","msg":"starting metrics server","path":"/metrics"}
	{"level":"info","ts":1600949264.2368062,"logger":"controller","msg":"Starting EventSource","reconcilerGroup":"keda.sh","reconcilerKind":"ScaledObject","controller":"scaledobject","source":"kind source: /, Kind="}
	{"level":"info","ts":1600949264.2368453,"logger":"controller","msg":"Starting EventSource","reconcilerGroup":"keda.sh","reconcilerKind":"ScaledJob","controller":"scaledjob","source":"kind source: /, Kind="}
	{"level":"error","ts":1600949268.53242,"logger":"controller-runtime.source","msg":"if kind is a CRD, it should be installed before calling Start","kind":"ScaledObject.keda.sh","error":"no matches for kind \"ScaledObject\" in version \"keda.sh/v1alpha1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/source/source.go:117\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:140\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:175\nsigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/manager/internal.go:661"}
	{"level":"error","ts":1600949272.8328192,"logger":"controller-runtime.source","msg":"if kind is a CRD, it should be installed before calling Start","kind":"ScaledJob.keda.sh","error":"no matches for kind \"ScaledJob\" in version \"keda.sh/v1alpha1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/source/source.go:117\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:140\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/internal/controller/controller.go:175\nsigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.2/pkg/manager/internal.go:661"}
	{"level":"error","ts":1600949272.8329563,"logger":"setup","msg":"problem running manager","error":"no matches for kind \"ScaledObject\" in version \"keda.sh/v1alpha1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nmain.main\n\t/workspace/main.go:119\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:204"}
	```

1. ...seemingly because it can't find the CRD:

	```
	$ kubectl get crd | grep keda
	scaledobjects.keda.k8s.io               2020-09-23T11:48:32Z
	triggerauthentications.keda.k8s.io      2020-09-23T11:48:32Z
	```

## Steps to Reproduce the Problem

Install the helm chart version keda-2.0.0-beta1.1.

**Comment by gldraphael** [comment]
This is like noticing a typo after sending an email --

The issue seems to be fixed in the keda-2.0.0-beta1.2 chart. I had to uninstall the old chart first for it to work:

```
$ helm uninstall -n keda keda
$ kubectl delete crd scaledobjects.keda.k8s.io triggerauthentications.keda.k8s.io
$ kubectl get crd | grep keda
scaledjobs.keda.sh                      2020-09-24T13:00:11Z
scaledobjects.keda.sh                   2020-09-24T13:00:11Z
triggerauthentications.keda.sh          2020-09-24T13:00:11Z

**Comment by zroubalik** [comment]
@gldraphael thanks for the info!
